CHANGELOG/CHANGELOG-1.11.md

```diff
@@ -0,0 +1,72 @@
+
+### Added
+
+- Add the following capabilities to Antrea-native policies:
+  * ClusterSet scoped policy rule is now compatible with `namespaces` field. ([#4571](https://github.com/antrea-io/antrea/pull/4571), [@Dyanngg])
+  * Layer 7 policy rule is now compatible with `enableLogging` field. ([#4625](https://github.com/antrea-io/antrea/pull/4625), [@qiyueyao])
+  * FQDN policy rule now supports clients using DNS over TCP. ([#4612](https://github.com/antrea-io/antrea/pull/4612), [@GraysonWu])
+- Add the following capabilities to the Multi-cluster feature:
+  * Support "noEncap", "hybrid", and "networkPolicyOnly" in-cluster traffic encapsulation modes with Multi-cluster Gateway. ([#4407](https://github.com/antrea-io/antrea/pull/4407), [@luolanzone])
+  * Support using WireGuard to encrypt cross-cluster traffic. ([#4606](https://github.com/antrea-io/antrea/pull/4606), [@hjiajing])
+- Add the following capabilities to the AntreaProxy feature:
+  * Graduate EndpointSlice from Alpha to Beta; the Antrea Agent now listens to EndpointSlice events by default. ([#4634](https://github.com/antrea-io/antrea/pull/4607), [@hongliangl])
+  * Support ProxyTerminatingEndpoints in AntreaProxy. ([#4607](https://github.com/antrea-io/antrea/pull/4607), [@hongliangl])
+  * Support rejecting requests to Services without available Endpoints. ([#4656](https://github.com/antrea-io/antrea/pull/4656), [@hongliangl])
+- Add the following capabilities to Egress policies:
+  * Support limiting number of Egress IPs that can be assigned to a Node via new configuration option `egress.maxEgressIPsPerNode` or Node annotation "node.antrea.io/max-egress-ips". ([#4593](https://github.com/antrea-io/antrea/pull/4593) [#4627](https://github.com/antrea-io/antrea/pull/4627), [@tnqn])
+  * Add `antctl get memberlist` CLI command to get memberlist state. ([#4611](https://github.com/antrea-io/antrea/pull/4611), [@Atish-iaf])
+
+### Changed
+
+- Ensure cni folders are created when starting antrea-agent with containerd on Windows. ([#4685](https://github.com/antrea-io/antrea/pull/4685), [@XinShuYang])
+- Decrease log verbosity value for antrea-agent specified in the Windows manifest for containerd from 4 to 0. ([#4676](https://github.com/antrea-io/antrea/pull/4676), [@XinShuYang])
+- Bump up cni and plugins libraries to v1.1.1. ([#4425](https://github.com/antrea-io/antrea/pull/4425), [@wenyingd])
+- Extend the message length limitation in the Conditions of Antrea-native policies to 256 characters. ([#4574](https://github.com/antrea-io/antrea/pull/4574), [@wenyingd])
+- Stop using ClusterFirstWithHostNet DNSPolicy for antrea-agent; revert it to the default value. ([#4548](https://github.com/antrea-io/antrea/pull/4548), [@antoninbas])
+- Ensure eventual Endpoint DNAT is performed in OVS for traffic to Multi-cluster Service when a local Service is selected. ([#4693](https://github.com/antrea-io/antrea/pull/4693), [@luolanzone])
```

Comment:
"Perform Service load balancing within OVS for Multi-cluster Service traffic, when the local member Service of the Multi-cluster Service is selected as the destination".

Reply:
done